lib/usd/ui/layerEditor/layerEditorWidget.cpp

```diff
@@ -229,20 +254,41 @@ QLayout* LayerEditorWidget::setupLayout_toolbar()
 
 void LayerEditorWidget::setupLayout()
 {
-    _treeView = new LayerTreeView(&_sessionState, this);
+    // Horizontal splitter that will contain the Layer Editor and Display Layer
+    // Contents Window.
+    auto mainHSplitter = new QSplitter(Qt::Horizontal);
```

Comment:
Added a new splitter to have the Layer Editor on the left and Layer Contents on the right.